src/prommis/leaching/tests/test_leach_flowsheet.py

```diff
@@ -1,19 +1,19 @@
-import pytest
-
 from pyomo.environ import (
-    assert_optimal_termination,
     SolverFactory,
     TransformationFactory,
+    assert_optimal_termination,
```

Comment:
Why is this file being edited here? These changes are just ordering of imports, and the original import order is more correct.